### ISSUE TITLE
Long kwargs (options) list is truncated

### ISSUE THREAD

**Issue by kumarshantanu** [martinklepsch/cljdoc#40]
When there are several options (destructured as a map) in a function signature, the options are truncated and only some of them are shown. Example: https://cljdoc.xyz/d/preflex/preflex/0.4.0/api/preflex.resilient (search for `:keys`).

**Comment by martinklepsch** [comment]
@kumarshantanu thanks for pointing this out, definitely should be improved.

I'm not 100% sure what should be done here though, without hinting at options, do you have thoughts on how to best handle this situation?

**Comment by kumarshantanu** [comment]
Pretty printing the argument list (in a monospaced font) could be a good, lossless option.

**Comment by martinklepsch** [comment]
[zprint]() returns the following for one of the larger argvecs, I guess this should bee good 🙂 

```clojure
(future-call-via thread-pool
                 {:keys [context-maker on-task-submit on-task-reject
                         on-task-error on-task-timeout],
                  :or {context-maker im/make-context,
                       on-task-submit in/nop,
                       on-task-reject (fn [_ _] (e/thread-pool-rejected)),
                       on-task-error (fn [_ e] (e/exception-occurred e)),
                       on-task-timeout (fn [_ _] (e/operation-timed-out))}}
                 f)
```

This is what `clojure.pprint/pprint` returns:

```clojure
(future-call-via
 thread-pool
 {:keys
  [context-maker
   on-task-submit
   on-task-reject
   on-task-error
   on-task-timeout],
  :or
  {context-maker im/make-context,
   on-task-submit in/nop,
   on-task-reject (fn [_ _] (e/thread-pool-rejected)),
   on-task-error (fn [_ e] (e/exception-occurred e)),
   on-task-timeout (fn [_ _] (e/operation-timed-out))}}
 f)
```

`node-cljfmt` acted a bit weird so didn't try this yet.

**Comment by martinklepsch** [comment]
@kumarshantanu If you would be up for providing a PR that runs zprint on argvecs longer than `n` characters, that would be fantastic 🥇

**Comment by kumarshantanu** [comment]
@martinklepsch Sure, I'll give it a shot. The `zprint` output looks cool.

**Comment by martinklepsch** [comment]
@kkinnear The default options seem fairly good, at the very least with the example I posted above. 

The code for argvec rendering is here, I assume it's mostly a matter of running zprint on the provided string, maybe skipping if it's length is below `N` characters.

https://github.com/martinklepsch/cljdoc/blob/f109f314a8ef519f66f30e0a12d6d5cbaa7d3839/src/cljdoc/renderers/html.clj#L105-L106

**Comment by kkinnear** [comment]
Thanks for the pointer to the code, it would have taken me a while to find that.

I tried implementing it just now, and it looks good!  I need to tune the width a bit, as there isn't quite 80 columns there apparently.  I'll see what I can do about generating a pull request tomorrow evening.   What fun!

**Comment by martinklepsch** [comment]
The code has been moved but you should be able to find it easily by searching for `def-code-block`.

Looking forward to having this, I think it will look awesome :)

**Comment by martinklepsch** [comment]
This has been fixed by #55, big thanks go to @kkinnear for this one.

An example can be seen here: https://cljdoc.xyz/d/preflex/preflex/0.4.0/api/preflex.resilient
